connectivity/manifests/echo-ingress-from-client.yaml

```diff
@@ -2,9 +2,9 @@ apiVersion: "cilium.io/v2"
 kind: CiliumNetworkPolicy
 metadata:
   namespace: cilium-test
-  name: echo-ingress-from-other-client
```

Comment:
The "other" here is intentional as the policy only allows from label `other=client` and only one of the clients has this label.